src/main/resources/org/broadinstitute/hellbender/tools/copynumber/cohort_determine_ploidy_and_depth.py

```diff
@@ -54,6 +54,12 @@
                    help="Output path to write posteriors")
 
 # optional arguments
+group.add_argument("--ploidy_correction_gq_threshold",
```

Comment:
Fun fact: argparse would convert the dashes to underscores in the object members automatically, allowing us to use snake-case. But leave it this way in solidarity